stories/Collapse/CollapseAccordionExample.js

```diff
@@ -44,4 +44,8 @@ const CollapseAccordionExample = () => {
   )
 }
 
-export default CollapseAccordionExample
+const CollapseAccordionExampleWithHook = props => (
+  <CollapseAccordionExample {...props} />
+)
+
+export default CollapseAccordionExampleWithHook
```

Comment:
This is not required. Can you revert it?
It breaks the `Source Code` block in the addon-info panel.